api/app/auto_spatial_advisory/zone_stats.py

```diff
@@ -54,11 +54,11 @@ def get_fuel_type_area_stats(grass_curing_date: date,
                              fuel_type_id: int,
                              area: float,
                              fuel_area: float):
-    # area is stored in square metres in DB. For user convenience, convert to hectares
```

Comment:
It turns out that using hectares was inconvenient. I've switched to meters and updated related code and tests.